datavalid.yml

```diff
@@ -301,6 +303,7 @@ schemas:
         description: >
           further description of the award
       - name: left_reason
+      - name: left_reason_desc
```

Comment:
What is the difference between `left_reason` and `left_reason_desc`? Would be nice if you leave a `description` for `left_reason_desc` at least.

Reply:
Thanks.